src/main/cpp/Robot.cpp

```diff
@@ -60,6 +63,22 @@ void Robot::SimulationInit() {}
 
 void Robot::SimulationPeriodic() {}
 
+/**
+ * @returns joystick value converted to meters per second
+ * TODO: change this to match the velocity calculations from 2022-Offseason
+**/
+double Robot::joy_val_to_mps(double val) {
+  return val*4;
+}
+
+/**
+ * @returns joystick value converted to radians per second
+ * TODO: change this to match the angular velocity calculations from 2022-Offseason
+**/
+double Robot::joy_rot_to_rps(double rot) {
+  return rot * 3*3*M_PI;
```

Comment:
Is this in radians per second or rotations per second? A max of 27 (rotations) or 4.5 (radians) rpm seems a bit high. random 3s are confusing.

Reply:
I actually don't remember how I came up with this. I'll try to change the joystick to mps and rps functions to better match what the calculations would be in 2022-Offseason & i'll improve the clarity/documentation

Reply:
In 2022-Offseason, joystick inputs are converted directly to volts (-1 to 1 range mapped to -12 to 12). To match this using velocity velocity, we'll need to know what the speed of the robot would be if the voltage applied to the drive motors is 12V. Until then (and because I doubt we'd ever want to go that fast anyway) I'll put guesses for linear and angular max speeds in the constants file.